libs/chiliui/chili/controls/object.lua

```diff
@@ -108,26 +108,21 @@ function Object:New(obj)
 
 	--// make an instance
 	for i, v in pairs(self) do --// `self` means the class here and not the instance!
-		if (i ~= "inherited") then
-			local t = type(v)
-			local ot = type(obj[i])
-			if (t == "table") or (t == "metatable") then
-				if (ot == "nil") then
-					obj[i] = {};
-					ot = "table";
-				end
-				if (ot ~= "table") and (ot ~= "metatable") then
-					Spring.Echo("Chili: " .. obj.name .. ": Wrong param type given to " .. i .. ": got " .. ot .. " expected table.")
-					obj[i] = {}
-				end
-
-				table.merge(obj[i], v)
-				if (t == "metatable") then
```

Comment:
It doesn't seem likely this branch does anything. How would `type(v)` be `"metatable"`?